package.json

```diff
@@ -141,6 +141,7 @@
     "start-bench": "run-p build-token watch-benchmarks start-server",
     "start-release": "run-s build-token build-prod-min build-css print-release-url start-server",
     "diff-tarball": "build/run-node build/diff-tarball && echo \"Please confirm the above is correct [y/n]? \"; read answer; if [ \"$answer\" = \"${answer#[Yy]}\" ]; then false; fi",
+    "publish-clean-check": "git clean -fdx && yarn install",
```

Comment:
nit: This step does not check anything. Consider renaming to a clearer name, like `prepare-for-publish`

Reply:
Updated in https://github.com/mapbox/mapbox-gl-js/pull/9590/commits/69b4df144c110dcef7cf1570f3140e0bd3f78d86.